### ISSUE TITLE
Clay Pipe Bee Drone insertion behaviour

### ISSUE THREAD

**Issue by mrnicerguy** [BuildCraft/BuildCraft#3386]
Recently I updated my Direwolf20 Server from version 1.2.1 to version 1.10.0. After upgrading i noticed that my alvearies and apiaries are missing Drones. I've automated them using Clay pipes to feed back the princesses and Drones. This worked well so that the Drone Slot was always filled with a full stack and the additional Drones just went in the next free inventory. This does not work anymore like this. The behaviour I now noticed is that only if the Drone Slot is empty there are Drones inserted. So they are not stacking up anymore in the Drone Slot until it is full and are then sent into the next free inventory.
My question here is, is this an intended behaviour or is the forestry compatibility broken in the recent version of buildcraft for 1.7.10.

Version Info:

Minecraft Version: 1.7.10
Buildcraft Version: 7.1.17
Buildcraft-Compat Version: 7.1.5
Forestry Version: 3.6.3.20

**Comment by AlexIIL** [comment]
This is *most likely* an issue on forestries side, rather than a BuildCraft issue. However you could try manually updating forestry to 3.6.12 (the latest 3.x forestry, as far as I can tell) and seeing if that helps.

**Comment by SandGrainOne** [comment]
Was Forestry a part of the update? I think there was a change to bees that changed how data is stored/represented. If I'm right, what you are trying to do should work with any new drones that are made.

**Comment by asiekierka** [comment]
Update your Forestry first. It's very old. Binnie's Mods is not an excuse;
use BinniePatcher.

2016-07-04 23:07 GMT+02:00 Terje H. <notifications@github.com>:

**Comment by mrnicerguy** [comment]
I will try that. Thanks for the suggestion.

**Comment by mrnicerguy** [comment]
Hi. I'm sorry for reporting late. ;-) I've managed to update the forestry version to 4.2.14.62. For this i had to update a couple of other mods and of course i had to use BinniePatcher. For this i had to switch my launcher to MultiMC because a newer forge version was needed for forestry. So in the end i got everything working and started testing. There were no stacking issues so far. So I thought that did the trick. For curiosity i switched back to the ftb launcher with the original recent modpack version because i didnt update anything on my server yet. I have to mention here that the server did run for a while with the recent direwolf20 modpack version. I've noticed that the stacking behavior returned to normal and then I finally understood what happened. After updating to the recent version of the modpack there were Bees in the alvearies and apiaries with old metadata. So after quite a plenty of cycles the Bee Drones with the old metadata were consumed and the new Drones stacked normally. That's the only explanation I can think of. So in the end there was no real issue, just a period of migration. I'm sorry for filing an issue prematurely. ;-) Nevertheless, thank you for your assistance.
